__tests__/app.test.js

```diff
@@ -298,3 +298,106 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/article_id", () => {
+  it("Patch 200: responds with the updated article where the votes count will have been altered correctly", () => {
+    const votes = { inc_votes: 13 };
+    return request(app)
+      .patch("/api/articles/7")
+      .send(votes)
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        console.log(article)
+        expect(article).toEqual({
+          article_id: 7,
+          title: "Z",
+          topic: "mitch",
+          author: "icellusedkars",
+          body: "I was hungry.",
+          created_at: "2020-01-07T14:08:00.000Z",
+          votes: 13,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  it("responds with a status 200: sends updated article when extra keys are sent on request body (they are ignored as long as inc_votes is present", () => {
+    const votes = {
+      inc_votes: -20,
+      username: "rogersop",
+      body: "I'm changing the votes",
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(votes)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 80,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  it("responds with an error 400 status when inc_votes is undefined (e.g. inc_votez - spelled incorrectly", () => {
```

Comment:
I think because of the test for ignoring additional keys this test should just say when inc_votes is missing and pass and empty object